When I ran this through Jenkins, I got:

> raise AssertionError('%s is not found in %s' % (needle, haystack))
> E AssertionError: en-us is not found in https://firefoxflicks.allizom.org/en-US/

Do we know where the lower-case "en-us" is coming from?

--- a/tests/test_file.py
+++ b/tests/test_file.py
@@ -34,6 +34,11 @@ class TestTemplate():
                 i += 1
                 main_page.change_locale(i)
                 selected = main_page.selected_lang
+                regex = re.search('((.*-)(.*))', selected)
+                try:
+                    selected = regex.group(2) + regex.group(3).upper()
+                except:
+                    print ""
                 Assert.contains(selected, main_page.get_url_current_page())
         else:
             print "There is no language selector on the page"

